service/biz/tpm_cert_verifier.go

```diff
@@ -93,6 +94,16 @@ func validateVerifyIakAndIDevIDCertsReq(req *VerifyIakAndIDevIDCertsReq) error {
 	return nil
 }
 
+func getCertSerialNumber(serial string) (string, error) {
+	// iakX509.Subject.SerialNumber comes in the format PID:xxxxxxx SN:1234JF
```

Comment:
As per our offline chat, I'm not very sure what the correct format is for this field in the spec. But LGTM for now to enable further testing.

Reply:
How about strings.Split(serial, "SN:")?

Then all you have to do is return sn[1] and make sure it has length of two.

Reply:
haha I considered this but then second guessed myself. Yeah Ok i'll change it to this